kolibri/core/content/serializers.py

```diff
@@ -437,7 +438,16 @@ class Meta:
 
     @property
     def channel_stats(self):
-        return self.context["channel_stats"]
+        channel_stats = self.context["channel_stats"]
+        if connection.vendor == "postgresql":
+            # Convert channel IDs to hex strings since they are stored as UUID
+            # objects when Kolibri connects to a PostgreSQL database
+            stats_convert_id = {}
+            for key in channel_stats.keys():
```

Comment:
is `channel_stats` always guaranteed to be a Dict? is there ever a chance it's a None?

Reply:
ah that's right! thank you for pointing this out. I will change that part